official/projects/cots_detector/crown_of_thorns_starfish_detection_pipeline.ipynb

```diff
@@ -64,7 +64,7 @@
     "id": "jDiIX2xawkJw"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.              # print('Skipping detection because it\'s close to the border.')
</pre><p>Can these commented-out print lines be replaced with logging.debug or similar?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/models/pull/10640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>